### DISCUSSION TITLE
Live app users table - unauthorized users

### DISCUSSION THREAD

**Discussion by applicationuser** [Budibase/budibase#10552] [Help]
Hello,

Noticed issue where live app users table has unauthorized users.  App displaying users that do not have access to app.  
app's dev users table shows valid users who have access to app. 

Budibase cloud , app version: 2.6.6

Dev app user table row count: 4
Live app user table row count:  11

sending screenshots to support email 

Thanks,

**Comment by melohagan** [comment]
Closing as discussed over email
